frontend/src/v5/ui/components/dashboard/dashboardProjectLayout/dashboardProjectLayout.component.tsx

```diff
@@ -22,14 +22,14 @@ import { AppBar } from '@components/shared/appBar';
 import { ProjectNavigation } from '@components/shared/navigationTabs';
 import { TeamspacesActionsDispatchers, ProjectsActionsDispatchers } from '@/v5/services/actionsDispatchers';
 import { DashboardParams } from '@/v5/ui/routes/routes.constants';
-import { Content } from './dashboardLayout.styles';
+import { Content } from './dashboardProjectLayout.styles';
 
-interface IDashboardLayout {
+interface IDashboardProjectLayout {
 	children: ReactNode;
 }
 
-export const DashboardLayout = ({ children }: IDashboardLayout): JSX.Element => {
-	const { teamspace, project, containerOrFederation } = useParams<DashboardParams>();
+export const DashboardProjectLayout = ({ children }: IDashboardProjectLayout): JSX.Element => {
```

Comment:
Isn't this more DashboardTeamspaceLayout? Since this is where you have the list of projects in the teamspace as well as teamspace settings/users/jobs?

Reply:
No, this is not that. It is one that as the containers/federations/issues and risks/etc. tabs
![image](https://github.com/3drepo/3drepo.io/assets/43547703/dfe8d444-f34f-42d6-a735-c34d733473a5)